src/st/en_thornweed_corpseweed.h

```diff
@@ -563,6 +563,8 @@ void EntityCorpseweed(Entity* self) {
             self->ext.corpseweed.stalkDoneGrowing = false;
             self->hitboxState = 0;
             self->drawFlags = FLAG_DRAW_ROTZ;
+            // Need a no-op self access here for PSP match
+            self->step_s;
```

Comment:
Don't think we have a way of determining what exactly, but there is a line here that accesses `self`.
My hypothesis would be something like `self->step++` that was supposed to increment but didn't and is a bug, but no way to know for sure. 

There are many things that will match here on PSP, found this via the permuter as `if (step && step) {}` empty statement, but I think something like this is a little clearer.

https://decomp.me/scratch/ZHUBn